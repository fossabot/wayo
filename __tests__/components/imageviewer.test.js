import 'babel-polyfill';
import { shallow,createLocalVue } from 'vue-test-utils';
import ImageViewer from '../../src/components/imageviewer';

const LocalVue = createLocalVue();

const Wrapper = shallow(ImageViewer, {
  localVue: LocalVue
});

describe('Imagepreviewer', () => {
  
});
