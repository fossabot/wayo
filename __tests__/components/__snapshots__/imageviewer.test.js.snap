// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot  1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "class": "wayo-imageviewer",
      "data-server-rendered": "true",
    },
    "children": Array [
      Object {
        "attribs": Object {
          "class": "wayo-imageviewer__mask",
          "style": "background-color:#000000;opacity:0.6;",
        },
        "children": Array [],
        "name": "div",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": Object {
          "data": " ",
          "next": Object {
            "data": "",
            "next": Object {
              "data": " ",
              "next": Object {
                "attribs": Object {
                  "class": "wayo-imageviewer__indicator",
                },
                "children": Array [
                  Object {
                    "data": "1/0",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "text",
                  },
                ],
                "name": "p",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": null,
                "parent": [Circular],
                "prev": [Circular],
                "type": "tag",
                "x-attribsNamespace": Object {
                  "class": undefined,
                },
                "x-attribsPrefix": Object {
                  "class": undefined,
                },
              },
              "parent": [Circular],
              "prev": [Circular],
              "type": "text",
            },
            "parent": [Circular],
            "prev": [Circular],
            "type": "comment",
          },
          "parent": [Circular],
          "prev": [Circular],
          "type": "text",
        },
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {
          "class": undefined,
          "style": undefined,
        },
        "x-attribsPrefix": Object {
          "class": undefined,
          "style": undefined,
        },
      },
      Object {
        "data": " ",
        "next": Object {
          "data": "",
          "next": Object {
            "data": " ",
            "next": Object {
              "attribs": Object {
                "class": "wayo-imageviewer__indicator",
              },
              "children": Array [
                Object {
                  "data": "1/0",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "text",
                },
              ],
              "name": "p",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": null,
              "parent": [Circular],
              "prev": [Circular],
              "type": "tag",
              "x-attribsNamespace": Object {
                "class": undefined,
              },
              "x-attribsPrefix": Object {
                "class": undefined,
              },
            },
            "parent": [Circular],
            "prev": [Circular],
            "type": "text",
          },
          "parent": [Circular],
          "prev": [Circular],
          "type": "comment",
        },
        "parent": [Circular],
        "prev": Object {
          "attribs": Object {
            "class": "wayo-imageviewer__mask",
            "style": "background-color:#000000;opacity:0.6;",
          },
          "children": Array [],
          "name": "div",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": [Circular],
          "parent": [Circular],
          "prev": null,
          "type": "tag",
          "x-attribsNamespace": Object {
            "class": undefined,
            "style": undefined,
          },
          "x-attribsPrefix": Object {
            "class": undefined,
            "style": undefined,
          },
        },
        "type": "text",
      },
      Object {
        "data": "",
        "next": Object {
          "data": " ",
          "next": Object {
            "attribs": Object {
              "class": "wayo-imageviewer__indicator",
            },
            "children": Array [
              Object {
                "data": "1/0",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "text",
              },
            ],
            "name": "p",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": null,
            "parent": [Circular],
            "prev": [Circular],
            "type": "tag",
            "x-attribsNamespace": Object {
              "class": undefined,
            },
            "x-attribsPrefix": Object {
              "class": undefined,
            },
          },
          "parent": [Circular],
          "prev": [Circular],
          "type": "text",
        },
        "parent": [Circular],
        "prev": Object {
          "data": " ",
          "next": [Circular],
          "parent": [Circular],
          "prev": Object {
            "attribs": Object {
              "class": "wayo-imageviewer__mask",
              "style": "background-color:#000000;opacity:0.6;",
            },
            "children": Array [],
            "name": "div",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": [Circular],
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {
              "class": undefined,
              "style": undefined,
            },
            "x-attribsPrefix": Object {
              "class": undefined,
              "style": undefined,
            },
          },
          "type": "text",
        },
        "type": "comment",
      },
      Object {
        "data": " ",
        "next": Object {
          "attribs": Object {
            "class": "wayo-imageviewer__indicator",
          },
          "children": Array [
            Object {
              "data": "1/0",
              "next": null,
              "parent": [Circular],
              "prev": null,
              "type": "text",
            },
          ],
          "name": "p",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": null,
          "parent": [Circular],
          "prev": [Circular],
          "type": "tag",
          "x-attribsNamespace": Object {
            "class": undefined,
          },
          "x-attribsPrefix": Object {
            "class": undefined,
          },
        },
        "parent": [Circular],
        "prev": Object {
          "data": "",
          "next": [Circular],
          "parent": [Circular],
          "prev": Object {
            "data": " ",
            "next": [Circular],
            "parent": [Circular],
            "prev": Object {
              "attribs": Object {
                "class": "wayo-imageviewer__mask",
                "style": "background-color:#000000;opacity:0.6;",
              },
              "children": Array [],
              "name": "div",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": [Circular],
              "parent": [Circular],
              "prev": null,
              "type": "tag",
              "x-attribsNamespace": Object {
                "class": undefined,
                "style": undefined,
              },
              "x-attribsPrefix": Object {
                "class": undefined,
                "style": undefined,
              },
            },
            "type": "text",
          },
          "type": "comment",
        },
        "type": "text",
      },
      Object {
        "attribs": Object {
          "class": "wayo-imageviewer__indicator",
        },
        "children": Array [
          Object {
            "data": "1/0",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "text",
          },
        ],
        "name": "p",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": null,
        "parent": [Circular],
        "prev": Object {
          "data": " ",
          "next": [Circular],
          "parent": [Circular],
          "prev": Object {
            "data": "",
            "next": [Circular],
            "parent": [Circular],
            "prev": Object {
              "data": " ",
              "next": [Circular],
              "parent": [Circular],
              "prev": Object {
                "attribs": Object {
                  "class": "wayo-imageviewer__mask",
                  "style": "background-color:#000000;opacity:0.6;",
                },
                "children": Array [],
                "name": "div",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": [Circular],
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "class": undefined,
                  "style": undefined,
                },
                "x-attribsPrefix": Object {
                  "class": undefined,
                  "style": undefined,
                },
              },
              "type": "text",
            },
            "type": "comment",
          },
          "type": "text",
        },
        "type": "tag",
        "x-attribsNamespace": Object {
          "class": undefined,
        },
        "x-attribsPrefix": Object {
          "class": undefined,
        },
      },
    ],
    "name": "div",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "class": undefined,
      "data-server-rendered": undefined,
    },
    "x-attribsPrefix": Object {
      "class": undefined,
      "data-server-rendered": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
